Dockerfile.build.template

```diff
@@ -21,18 +21,19 @@ RUN mkdir -p rootfs-overlay && \
 
 COPY package.json /usr/src/app/
 
-# Install only the production modules
+# Install only the production modules that have C extensions
 RUN JOBS=MAX npm install --production --no-optional --unsafe-perm \
-	&& npm dedupe
+	&& npm dedupe && cp -R node_modules node_modules_prod
```

Comment:
It makes more sense imo to move the `cp -R` to the `RUN npm install` below, rather than persisting the copy in this layer